### DISCUSSION TITLE
Hook does not collect the values of dropdown select menu

### DISCUSSION THREAD

**Discussion by karadagde** [react-hook-form/react-hook-form#6685] [Q&A] [answered]
Hi,

I am sorry if my question is silly. I have this form: https://codesandbox.io/s/suspicious-chaum-mc007

When I click submit, I can view the values of my input fields but the the values of dropdown menu. I created this form on my computer and I was having above issue. When I copied the same code to codesandbox, it was working properly.  So, now I know that code is fine, what might be the problem that I am having when I am running it on localhost?

These are my dependencies:
"dependencies": {
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.3.2",
    "@testing-library/user-event": "^7.1.2",
    "axios": "^0.19.2",
    "bootstrap": "^4.4.1",
    "moment": "^2.29.1",
    "react": "^16.13.0",
    "react-bootstrap": "^1.0.0-beta.17",
    "react-dom": "^16.13.0",
    "react-hook-form": "^7.16.1",
    "react-redux": "^7.2.0",
    "react-router-dom": "^5.1.2",
    "react-scripts": "3.4.0",
    "redux": "^4.0.5",
    "redux-thunk": "^2.3.0"

**Comment by thanh-nguyen-95** [comment, marked as answer]
Is https://codesandbox.io/s/suspicious-chaum-mc007 the correct url because I don't see any form in it. Please check again.

I notice the CodeSandBox is using the latest version of `react` and `react-hook-form` while your local project isn't. That could be a problem.

You can also try to remove `node_modules` and `package-lock.json` or `yarn.lock` and reinstall everything. It can sometimes solve the problem (or create more problems lol 😁).

Good luck.
